extensions/ql-vscode/src/view/data-flow-paths/__tests__/DataFlowPathsView.spec.tsx

```diff
@@ -17,8 +19,16 @@ describe(DataFlowPathsView.name, () => {
   });
 
   it("renders a data flow paths view", () => {
-    render({ dataFlowPaths: createMockDataFlowPaths() });
+    const dataFlowPaths: DataFlowPaths = {
+      codeFlows: createMockCodeFlows(),
+      ruleDescription: "Rule description",
+      message: createMockAnalysisMessage(),
+      severity: "Recommendation",
```

Comment:
Could we add a parameter to `createMockDataFlowPaths` that overrides just some of the properties, like we've done for `createMockVariantAnalysis`? That would make it easier to add new properties to the `DataFlowPaths` interface without updating all places where we're trying to override some of these properties.